apiary.apib

```diff
@@ -0,0 +1,108 @@
+FORMAT: 1A
+
+# Defence Request Rota Internal API
+Defence Request Rota App Internal API's
+
+## Authorization
+Access tokens are obtained from the Auth app using OAuth
+
+Access tokens must be passed for each request in an Authorization header.
+
+A missing Authorization Header or invalid Authorization Token value for a protected endpoint will result in a 401 response
+
+# Group API Root [/]
+
+This resource does not have any attributes. Instead it offers the initial API affordances in the form of the links in the JSON body.
+
+It is recommended to follow the “url” link values, [Link](https://tools.ietf.org/html/rfc5988) or Location headers where applicable to retrieve resources. Instead of constructing your own URLs, to keep your client decoupled from implementation details. We will use link values for all collection responses.
+
+## Retrieve the Entry Point [GET]
+
++ Response 200 (application/json)
+
+        {
+            "schemes_url": "/api/v1/schemes"
+        }
+
+# Group Schemes
+Schemes contain a List of Solicitors that are on Duty
+
+## Schemes Collection [/api/v1/schemes]
```

Comment:
Do we need to be able to filter this list? How many schemes are there across the country?